src/custom_modules/DAO_modules/donations.ts

```diff
@@ -158,6 +158,37 @@ async function getAll(sort, page, limit = 10, filter = null) {
   }
 }
 
+async function getTransactionCostsReport() {
+    let [res] = await DAO.query(`
+    SELECT count(ID) as donationsCount,
+      (SELECT round(sum(transaction_cost)) FROM Donations
+        WHERE YEAR(timestamp_confirmed) = YEAR(CURRENT_DATE)
+        AND MONTH(timestamp_confirmed) = MONTH(CURRENT_DATE) - 1) as costPrevMonth,
+      (SELECT round(sum(transaction_cost)) FROM Donations
+        WHERE YEAR(timestamp_confirmed) = YEAR(CURRENT_DATE) - 1
+        AND MONTH(timestamp_confirmed) = MONTH(CURRENT_DATE) - 1) as costPrevMonthPrevYear,
```

Comment:
`BETWEEN LAST_DAY(CURRENT_TIMESTAMP - INTERVAL 1 YEAR - INTERVAL 2 MONTH) + INTERVAL 1 DAY AND LAST_DAY(CURRENT_TIMESTAMP - INTERVAL 1 YEAR - INTERVAL 1 MONTH)`